config/initializers/datadog.rb

```diff
@@ -2,7 +2,6 @@
   Datadog.configure do |c|
     c.use :rails, service_name: 'rails'
     c.use :delayed_job, service_name: 'delayed_job'
-    c.use :dalli, service_name: 'memcached'
```

Comment:
This made me create https://github.com/openfoodfoundation/openfoodnetwork/issues/7703.